frontend/app/component/form/polling_station_choice/PollingStationChoiceForm.test.tsx

```diff
@@ -22,13 +23,28 @@ describe("Test PollingStationChoiceForm", () => {
     await user.type(pollingStation, "1234567");
     expect(pollingStation).toHaveValue("123456");
 
-    expect(screen.getByText("Kies het stembureau")).not.toBeVisible();
+    await user.click(submitButton);
+  });
+
+  test("Polling station list", async () => {
+    const user = userEvent.setup();
+
+    render(
+      <PollingStationProvider electionId={1}>
+        <PollingStationChoiceForm />
+      </PollingStationProvider>,
+    );
 
+    expect(screen.getByText("Kies het stembureau")).not.toBeVisible();
     const openPollingStationList = screen.getByTestId("openPollingStationList");
     await user.click(openPollingStationList);
 
     expect(screen.getByText("Kies het stembureau")).toBeVisible();
 
-    await user.click(submitButton);
+    // Check if the station number and name exist and are visible
+    expect(screen.getByText("20")).toBeVisible();
+    expect(screen.getByText('Stembureau "Op Rolletjes"')).toBeVisible();
+    expect(screen.getByText("21")).toBeVisible();
+    expect(screen.getByText("Testplek")).toBeVisible();
```

Comment:
```suggestion
    const pollingStationList = screen.getByTestId("polling_station_list")
    expect(within(pollingStationList).getByText("20")).toBeVisible();
    expect(within(pollingStationList).getByText('Stembureau "Op Rolletjes"')).toBeVisible();
    expect(within(pollingStationList).getByText("21")).toBeVisible();
    expect(within(pollingStationList).getByText("Testplek")).toBeVisible();
```

Reply:
Suggestion above does not include the required import of `within`.

Reply:
Nice, thanks